e2e/protractor-ci.conf.js

```diff
@@ -5,6 +5,10 @@ config.capabilities = {
   chromeOptions: {
     args: ['--headless', '--no-sandbox'],
   },
+  chromeDriver:
+    process.platform === 'win32'
+      ? './node_modules/webdriver-manager/selenium/chromedriver_89.0.4389.114.exe'
+      : './node_modules/webdriver-manager/selenium/chromedriver_89.0.4389.114',
```

Comment:
Implemented as a temporary workaround due to version mismatch of chromedriver in E2E test.